backend/.editorconfig

```diff
@@ -107,6 +107,15 @@ dotnet_diagnostic.SA1208.severity = warning
 dotnet_diagnostic.SA1623.severity = none
```

Comment:
NOTE TO REVIEWERS: there are some changes included here that were handled by automation. As a result, I have marked the files that I actually changed with a comment. Other files can be ignored.